egs2/aphasiabank/asr1/conf/tuning/train_asr_ebranchformer_small_wavlm_large1_interctc9.yaml

```diff
@@ -0,0 +1,115 @@
+# Uses 2 V100 (32GB)
```

Comment:
Since the configs and results with inter ctc are highly similar, maybe we can consider simplify the PR by merging some of them?

Reply:
I'll remove the interctc9 config as it's mostly the same as interctc6.
But for other configs I think maybe it's best to keep them for easier reproducibility.